apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -211,3 +211,11 @@ function load_blog_posts_block() {
 	require_once __DIR__ . '/blog-posts-block/index.php';
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_blog_posts_block' );
+
+/**
+ * Load WPCOM Block Editor NUX
+ */
+function load_wpcom_block_editor_nux() {
+	require_once __DIR__ . '/wpcom-block-editor-nux/class-wpcom-block-editor-nux.php';
+}
+add_action( 'plugins_loaded', __NAMESPACE__ . '\load_wpcom_block_editor_nux' );
```

Comment:
Related to my other comment, we could temporarily comment this action until Dotcom side of things is ready.